src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/ExceptionUtils.vb

```diff
@@ -174,11 +174,11 @@ Namespace Microsoft.VisualBasic.CompilerServices
         Friend Const DISP_E_NOTACOLLECTION As Integer = &H80020011I
 
         Friend Shared Function VbMakeIllegalForException() As System.Exception
-            Return VbMakeExceptionEx(vbErrors.IllegalFor, GetResourceString(SR.ID92)) ' 92 - IllegaFor
+            Return VbMakeExceptionEx(vbErrors.IllegalFor, SR.Format(SR.ID92)) ' 92 - IllegaFor
```

Comment:
```suggestion
            Return VbMakeExceptionEx(vbErrors.IllegalFor, SR.ID92) ' 92 - IllegaFor
```

`SR.Format` is needed only when there is formatting needed, ie there is one or more additional arguments.